pkg/asset/machines/openstack/machines.go

```diff
@@ -46,6 +46,11 @@ func Machines(clusterID string, config *types.InstallConfig, pool *types.Machine
 		return nil, err
 	}
 
+	volumeAZs := []string{""}
```

Comment:
Can we add the default `[]string{""}` value for `RootVolume.Zones` to `defaultOpenStackMachinePoolPlatform` so that we don't have to have this code here?

Reply:
I think we can't... `RootVolume` is an optional parameter for machine pool, and it's not enabled by default (in other words it is `nil` in `defaultOpenStackMachinePoolPlatform`). If we add `[]string{""}` value for `RootVolume.Zones`, it means that the default would be:

```go
openstacktypes.MachinePool{
		Zones: []string{""},
		RootVolume: &openstacktypes.RootVolume{
			Zones: []string{""},
		},
	}
```

And we always create the RootVolume even if it's not required.

Reply:
Instead, I added the default value in "github.com/openshift/installer/pkg/types/openstack/defaults" which imo is more appropriate.

Reply:
Ah, that makes sense about not being able to set it in `defaultOpenStackMachinePoolPlatform`. It would be nice to be able to set the default value when the install config defaults are set then, but I won't hold up this PR for that.